hibernate-core/src/main/java/org/hibernate/dialect/SAPDBDialect.java

```diff
@@ -195,23 +198,18 @@ public CaseFragment createCaseFragment() {
 	public SqmMultiTableMutationStrategy getFallbackSqmMutationStrategy(
 			EntityMappingType rootEntityDescriptor,
 			RuntimeModelCreationContext runtimeModelCreationContext) {
-		throw new NotYetImplementedFor6Exception( getClass() );
-
-//		return new LocalTemporaryTableBulkIdStrategy(
-//				new IdTableSupportStandardImpl() {
-//					@Override
-//					public String generateIdTableName(String baseName) {
-//						return "temp." + super.generateIdTableName( baseName );
-//					}
-//
-//					@Override
-//					public String getCreateIdTableStatementOptions() {
-//						return "ignore rollback";
-//					}
-//				},
-//				AfterUseAction.DROP,
-//				null
-//		);
+		return new LocalTemporaryTableStrategy(
+				new IdTable( rootEntityDescriptor, name -> "temp." + name ),
+				() -> new PhysicalIdTableExporter() {
```

Comment:
Is it correct to use `PhysicalIdTableExporter` here, even though it's really a temp table?

Reply:
No, it should use `TempIdTableExporter`

Reply:
OK thanks I will fix that.

Reply:
Done.